packages/theme/pages/Home.vue

```diff
@@ -193,6 +194,9 @@ export default {
       }
     ];
     const { search, result } = useFacet();
+    const context = useVSFContext();
+    const req = useProducts(context);
+    console.log(req);
```

Comment:
If you have tested it and it works, please remove this code. UseProducts is not used by default in the app so we will add the docs explaining how to use it instead :)